### PR TITLE
[Issue #101] Updated creation of submissions for all users

### PR DESCRIPTION
**Pull request by mohameddahrouj**

### CODE REVIEW

File: src/main/java/com/carleton/CapstoneSystem/restfulControllers/ProjectResource.java [hunk 1]
```diff
@@ -72,4 +72,10 @@ public Response addStudents(@RequestBody ProjectDTO projectDTO) {
 
         return projectController.addStudents(projectDTO);
     }
+
+    @PostMapping("/addDeliverable")
```
Comment (tosinoni) [inline src/main/java/com/carleton/CapstoneSystem/restfulControllers/ProjectResource.java]: I think we need a new resource called SubmissionResource for this endpoint. The endpoint should take a submissionDTO. The submissionDTO has name, dueDate and projectDTO.

File: src/main/java/com/carleton/CapstoneSystem/Controllers/ProjectController.java [hunk 2]
```diff
@@ -187,6 +187,16 @@ public Response addStudents(ProjectDTO projectDTO){
         return  Response.status(Response.Status.OK).entity(projectMembers).build();
     }
 
+    public Response addDeliverable(ProjectDTO projectDTO){
+        if(projectDTO == null) {
```
Comment (tosinoni) [inline src/main/java/com/carleton/CapstoneSystem/Controllers/ProjectController.java]: This should be in a new controller called SubmissionController.

File: src/main/resources/static/index.html [hunk 3]
```diff
@@ -27,7 +27,7 @@
                         <span class="icon-bar"></span>
                         <span class="icon-bar"></span>
                     </button>
-                    <a class="navbar-brand" href="/welcome">Capstone System</a>
+                    <a class="navbar-brand" href="/welcome">My Kapi</a>
                 </div>
```
Comment (tosinoni) [inline src/main/resources/static/index.html]: Sweet 👍

File: src/main/resources/static/js/projects/project-controller.js [hunk 4]
```diff
@@ -97,6 +97,22 @@ angular.module('CapstoneSystem')
             });
         }
 
+        $scope.openAddDeliverableToProjectModal = function () {
```
Comment (tosinoni) [inline src/main/resources/static/js/projects/project-controller.js]: Perfect

File: src/main/java/com/carleton/CapstoneSystem/Controllers/ProjectController.java [hunk 5]
```diff
@@ -187,6 +187,12 @@ public Response addStudents(ProjectDTO projectDTO){
         return  Response.status(Response.Status.OK).entity(projectMembers).build();
     }
 
+    public Response addDeliverable(ProjectDTO projectDTO){
```
Comment (tosinoni) [inline src/main/java/com/carleton/CapstoneSystem/Controllers/ProjectController.java]: Need to have a submission controller. This class is already doing a lot.

**Comment by tosinoni** [comment]
Good work bro. Nice client to server integration.

File: src/main/java/com/carleton/CapstoneSystem/restfulControllers/ProjectResource.java (hunk 1 above)
Comment (mohameddahrouj) [reply src/main/java/com/carleton/CapstoneSystem/restfulControllers/ProjectResource.java]: This PR is for the front end, I will leave that up to you or Ali to decide.

File: src/main/java/com/carleton/CapstoneSystem/Controllers/ProjectController.java (hunk 5 above)
Comment (mohameddahrouj) [reply src/main/java/com/carleton/CapstoneSystem/Controllers/ProjectController.java]: I agree, it should be decoupled, just wanted to create a place holder.

Comment (tosinoni) [reply src/main/java/com/carleton/CapstoneSystem/Controllers/ProjectController.java]: good idea.

File: src/main/java/com/carleton/CapstoneSystem/restfulControllers/ProjectResource.java (hunk 1 above)
Comment (tosinoni) [reply src/main/java/com/carleton/CapstoneSystem/restfulControllers/ProjectResource.java]: sounds good man 😄

**Pull request merged**